src/cursor.js

```diff
@@ -10,8 +10,21 @@
 import {valToKeyPath, makeCursor} from './utils';
 import createAtom from 'atom-store';
 
+const inMemory = (initial, transition) => {
+  let rootState = initial;
+  const read = () => rootState;
+  const write = (fn, ...context) => {
+    const oldState = read();
+    const newState = fn(oldState);
+    transition(newState, oldState, ...context);
```

Comment:
Agreed :) After https://github.com/jameshopkins/atom-store/pull/2 is merged I plan to make a atom-store PR for both `context` and the arg order swap